qpmodel/Statis.cs

```diff
@@ -102,7 +102,7 @@ int whichBucket(Value val)
             if (val is DateTime)
                 frac = (double)((dval - dlb).Divide(dub - dlb));
             else
-                frac = (double)(dval - dlb) / (dub - dlb);
```

Comment:
> the join estimation have to be changed back to previous due to q95 of
tpcds not working: the node tree have a cycle.

What's that problem？Can you put an issue tracking it?